client/src/navigation/RouterConfig.tsx

```diff
@@ -21,17 +24,25 @@ const EventAdmin = React.lazy(() => import('./EventAdminContainer'));
 //                        url     label   component
 export type RouteTuple = [string, string, React.ComponentType<{}>];
 export type RouteList = Array<RouteTuple>;
+export const registerUrl = '/events/:eventId/register';
 
 const RouterConfig = () => {
   const { pathname } = useLocation();
 
   return (
     <Switch>
       <Redirect from="/:url*(/+)" to={pathname.slice(0, -1)} />
-      <Route exact path="/" component={Default} />
-      <Route exact path="/events/:eventId/register" component={RegisterPage} />
+      <Provider store={regStore}>
+        <Route exact path="/" component={Default} />
+        <Route exact path={registerUrl}>
+          <Redirect to={`${pathname}/form${window.location.search}`} />
+        </Route>
+        <Route exact path={`${registerUrl}/form`} component={RegisterStep} />
+        <Route exact path={`${registerUrl}/payment`} component={PaymentStep} />
+        <Route exact path={`${registerUrl}/finished`} component={FinalStep} />
```

Comment:
This sets us up nicely to easily add other routes for things, like supplemental payments.